packages/react-native/React/Views/RCTConvert+Transform.m

```diff
@@ -144,6 +144,122 @@ + (CATransform3D)CATransform3D:(id)json
   return transform;
 }
 
++ (CATransform3D)CATransform3D:(id)json withWidth:(CGFloat)width height:(CGFloat)height {
```

Comment:
Can we have `[RCTConvert CATransform3D:]` call `[RCTConvert CATransform3D:json withWidth:NaN height:NaN]` or something similar? We don't want to be duplicating this code.

Any caller to the other CATransform should error or fallback to zero when width/height are not provided.

Reply:
Makes sense, thanks. Added an error log and returned 0 when height/width is not provided for percentage values.